.github/workflows/python_unit_tests.yml

```diff
@@ -10,9 +10,13 @@ jobs:
     - name: Checkout covid-data-model
       uses: actions/checkout@v2
     - name: Run Black
-      uses: lgeiger/black-action@v1.0.1
+      uses: RojerGS/python-black-check@1.0.1
```

Comment:
Why not run black in the same way that `make test` is run so it more closely replicates what happens on our local workstations?

Reply:
I do like that idea - the main reason was for speed, the environment needed is a bit heavy (though many ways around that problem) - definitely down to address in the future though